config/config.go

```diff
@@ -387,7 +387,7 @@ type Configuration struct {
 	ConsumerPoolSize    int                        `json:"consumer_pool_size" envconfig:"CONVOY_CONSUMER_POOL_SIZE"`
 	EnableProfiling     bool                       `json:"enable_profiling" envconfig:"CONVOY_ENABLE_PROFILING"`
 	Metrics             MetricsConfiguration       `json:"metrics" envconfig:"CONVOY_METRICS"`
-	InstanceIngestRate  int                        `json:"pub_sub_ingest_rate" envconfig:"CONVOY_PUB_SUB_INGEST_RATE"`
+	InstanceIngestRate  int                        `json:"instance_ingest_rate" envconfig:"CONVOY_INSTANCE_INGEST_RATE"`
```

Comment:
This will break things in cloud

Reply:
We can change it before it gets to the cloud; I don't see it being an issue.

Reply:
I made the change in my rate-limiter PR, but some other PR reverted it that's why I made it here again.

Reply:
It's mainly because of [this issue](https://github.com/jmoiron/sqlx/issues/196)

Reply:
Okay